src/services/ui/src/components/UsaBanner/index.tsx

```diff
@@ -59,6 +63,16 @@ export const UsaBanner = () => {
           </div>
         </button>
       )}
+      {role && userContext?.user  &&(
```

Comment:
This condition is a bit confusing because at first glance it looks like a user would have a role with access but we're showing a banner communicating they don't have access. Something like `{ !hasAccess && ( ... )}` would represent the intent a lot better